src/components/Comments.jsx

```diff
@@ -21,7 +20,7 @@ function Comments() {
     return (
         <section>
             <ul className='comments-box'> 
-                {allComments.map((comment) => {
+                {allComments.length === 0 ? <p> no comments on this article </p> :allComments.map((comment) => {
                     return <li className='comments' key={comment.comment_id}> 
                            {comment.body}; 
```

Comment:
Was the ; after {comment.body} supposed to be left in there? It might be a style choice you wanted to make but just wanted to make sure as at the moment it will show at the end of every comment. You don't need it just because it's the end of a line in our code :)